allennlp/models/ensemble/bidaf_ensemble.py

```diff
@@ -0,0 +1,155 @@
+from typing import Dict, List, Any, Tuple
+
+from overrides import overrides
+import torch
+
+from allennlp.common.checks import ConfigurationError
+from allennlp.models.ensemble.ensemble import Ensemble
+from allennlp.models.archival import load_archive
+from allennlp.models.model import Model
+from allennlp.models.reading_comprehension import BidirectionalAttentionFlow
+from allennlp.common import Params
+from allennlp.data import Vocabulary
+from allennlp.training.metrics import SquadEmAndF1
+
+@Model.register("bidaf-ensemble")
+class BidafEnsemble(Ensemble):
+    """
+    This class ensembles the output from multiple BiDAF models.
+
+    It combines results from the submodels by taking the option with the most votes and breaking ties with
+    the average confidence of the start and stop indices.
```

Comment:
What happens if you just use average confidence of start and stop indices?  This is the typical thing to do, and given that you already have the logic, I'd recommend just removing the voting and using the confidence averaging as the main (only) code path.

This would also make it quite trivial to include a loss, and make this whole thing trainable, if we ever wanted to do that.

Reply:
I can try this.  I did what Chris Clarke ended up doing after trying different approaches.